### ISSUE TITLE
Disabling position fixed

### ISSUE THREAD

**Issue by lewis150193** [Mastermindzh/react-cookie-consent#46]
Hey Mastermindzh, 

I have been trying to disable to position='fixed' CSS in styles, I have overwritten some styles to my needs but the position='fixed' seems to always come back when it isn't needed, have you run into this issue or found any workaround?

**Comment by Mastermindzh** [comment]
This is a bug.

The problem is I overwrite whatever position you set if you pass a location of TOP or BOTTOM.

Problem is... by default the location is set to "BOTTOM".

To fix this, for now, you can pass something random to the "location" property and it will fix your issue:

```js
  <CookieConsent
    location="static"
  >
```

I'll submit a new version with this bug fixed in the next few minutes.

**Comment by Mastermindzh** [comment]
The bug is fixed in [2.4.0](https://www.npmjs.com/package/react-cookie-consent/v/2.4.0)

Now when you override the position using css it'll do what you say:

```js
  <CookieConsent
	style={{position: "static"}}
  >
```

[Demo on StackBlitz](https://stackblitz.com/edit/react-cookie-consent-decline-button-demo-sfb1ri)

Don't forget to star if you like the project!

Have a nice day.

**Comment by lewis150193** [comment]
Thanks for the help!
